smac/smbo/smbo.py

```diff
@@ -204,13 +204,14 @@ def choose_next(self, X, Y,
             else:
                 num_configurations_by_local_search = 10
 
-        # initial SLS by incumbent +
-        # best configuration from next_configs_by_random_search_sorted
+        # initiate local search with best configurations from previous runs
+        configs_previous_runs = self.runhistory.get_all_configs()
+        configs_previous_runs_sorted = self._get_acq_values_sorted(configs_previous_runs)
+        num_configs_local_search = min(len(configs_previous_runs_sorted), num_configurations_by_local_search)
         next_configs_by_local_search = \
             self._get_next_by_local_search(
-                [self.incumbent] +
                 list(map(lambda x: x[1],
-                         next_configs_by_random_search_sorted[:num_configurations_by_local_search - 1])))
+                         configs_previous_runs_sorted[:num_configs_local_search])))
```

Comment:
Because the incumbent was also added. So the number of configurations for local search was 10, you did local search on the incumbent + 9 random configurations.